plugins/dashboards/frontend/public/stylesheets/main.scss

```diff
@@ -33,6 +33,16 @@
 			right:16px;
 		}
 	}
+
+    .notes-link-ellipsis {
```

Comment:
This goes within .clyd-widget, as per bem conventions
Please check the example of vis-table
https://github.com/Countly/countly-server/blob/0d4a7a2afc9304ab6abcd79539ee1a30e2fe9700/plugins/dashboards/frontend/public/stylesheets/main.scss#L189